lib/caps/ICL/iHD

```diff
@@ -0,0 +1,78 @@
+###
+### Copyright (C) 2019 Intel Corporation
+###
+### SPDX-License-Identifier: BSD-3-Clause
+###
+
+###
+### kate: syntax python;
+###
+
+# https://github.com/intel/media-driver/blob/master/docs/media_features.md
```

Comment:
we have many duplicates here. is it possible to reuse configurations from the previous Gen? like
```
import  GLK/iHD
caps.decode.jjpeg.maxres=res16k
```
If someone does need full details, we can add some command line to print full configuration.

Reply:
I think I'd prefer not to inherit any settings here.  I'd like to be able to see all the explicit capabilities of a driver/plat by viewing a single file without needing to follow the import trail or run the code with an option.

If we inherit caps from other caps, then all caps would need to be loaded at runtime and require explicit copying of inherited caps before modifying current caps.  Also, we would need to convert these directories/files into Python modules which I prefer we don't.

Reply:
ok, thanks for explaining.